src/store/wizardSlice.ts

```diff
@@ -43,6 +44,18 @@ export type RegistrationType =
 
 export type ComplianceType = 'openscap' | 'compliance';
 
+export type UserWithAdditionalInfo = Required<User>;
```

Comment:
I just realized that the User type also allows the fields to be `undefined`. Since we will be using `''` as the empty state for the form fields, we can tighten this up even a bit more by using the `NonNullable` utility function in addition to `Required`.

https://www.typescriptlang.org/docs/handbook/utility-types.html#nonnullabletype

Reply:
![image](https://github.com/user-attachments/assets/24fdc8e4-945e-47bd-8d69-415ef229bff6)